xcode/Subconscious/Shared/Components/Common/Transclude/TranscludeButtonStyle.swift

```diff
@@ -12,19 +12,33 @@ struct TranscludeButtonStyle: ButtonStyle {
         cornerRadius: AppTheme.cornerRadiusLg
     )
     @Environment(\.isEnabled) private var isEnabled
-
+    @Environment(\.colorScheme) var colorScheme
+    
+    func background(_ configuration: Configuration) -> Color {
+        switch (colorScheme) {
+        case .dark:
+            return configuration.isPressed
+                ? Color.backgroundPressed
+                : Color.secondaryBackground
```

Comment:
Is tertiary too light for dark mode? Is secondary too dark for light mode?

Would be nice if we could unify these, but not strictly necessary.